src/features/export/export-menu.tsx

```
@@ -136,58 +144,67 @@ export const ExportMenu = ({ animationMode }: ExportMenuProps = {}) => {
+            pixelRatio: 2,
+            backgroundColor: "#fff",
+          });
+          fileName = `${name}.jpg`;
+          break;
+
+        case "SVG":
+          // Convert to PNG first (which works reliably), then wrap in SVG
+          // This avoids the known issues with html-to-image's toSvg function
+          const pngDataUrl = await toPng(freshEditor!, {
+            pixelRatio: 2,
+          });
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)